pkg/datamover/dataupload_delete_action.go

```diff
@@ -57,6 +57,8 @@ func genConfigmap(bak *velerov1.Backup, du velerov2alpha1.DataUpload) *corev1api
 		BackupStorageLocation: bak.Spec.StorageLocation,
 		SnapshotID:            du.Status.SnapshotID,
 		RepositoryType:        GetUploaderType(du.Spec.DataMover),
```

Comment:
Although this line is unrelated to this PR change, this line does not look right. 
It should be something like "kopia".

Reply:
Yes, I have modified it, thanks.